package/src/components/CartCheckoutButton/v1/CartCheckoutButton.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+
+class CartCheckoutButton extends Component {
+  static propTypes = {
+    /**
+     * Text to display inside the button
+     */
+    buttonText: PropTypes.string,
+    /**
+     * On object of component children to pass into this component
+     */
+    components: PropTypes.shape({
+      Button: PropTypes.oneOfType([PropTypes.string, PropTypes.func])
+    }),
+    /**
+     * Set to `true` to prevent the button from calling `onClick` when clicked
+     */
+    isDisabled: PropTypes.bool,
+    /**
+     * Onclick function to pass to the Button component. Not handled internally, directly passed
+     */
+    onClick: PropTypes.func.isRequired
+  }
+
+  static defaultProps = {
+    buttonText: "Checkout"
```

Comment:
Need to add `onClick() {}` as a no-op function since we're not checking for it before using it in the `handleOnClick` method. [example from TextInput](https://github.com/reactioncommerce/reaction-component-library/blob/4155d4583c83e00dc0bfa77614901bc0654eb6eb/package/src/components/TextInput/v1/TextInput.js#L287)

Reply:
I initially had it there, but after a discussion with @willopez we thought that if the onClick was required, we wouldn't need to provide a default since there would have to be one provided. Do we still need it even if there is a default?

Reply:
ah that's true since `onClick` is `required` we probably don't need the no-op function in the `defaultProps`